app/Http/Controllers/WishlistController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Wishlist;
+
+class WishlistController extends Controller
+{
+    public function wishlist() 
+    {
+        return view('wishlist', ['games' => self::getWishlist()]);
+    }
+
+    private function getWishlist() 
+    {
+        return 
+        [
+            ['title'=>'Age of Empires 3'],
+            ['title'=>'Simcopter'],
+            ['title'=>'Darkest Dungeon'],
```

Comment:
just to clarify, this site is about board games, not video games :)

Reply:
Ah, yes. My brain defaulted to video games. Will update with board games for relevancy :)